sys/include/net/gcoap.h

```diff
@@ -618,10 +618,13 @@ typedef ssize_t (*gcoap_link_encoder_t)(const coap_resource_t *resource, char *b
 
 /**
  * @brief   A modular collection of resources for a server
+ * @note    Resources must be sorted.
  */
 typedef struct gcoap_listener {
     const coap_resource_t *resources;   /**< First element in the array of
-                                         *   resources; must order alphabetically */
+                                             resources; must order by the ASCII
```

Comment:
I know the original wording is the same, but
```suggestion
                                             resources; must be ordered by the ASCII
```